lib/active_remote/typecasting/boolean_typecaster.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveRemote
+  module Typecasting
+    class BooleanTypecaster
+       FALSE_VALUES = ["n", "N", "no", "No", "NO", "false", "False", "FALSE", "off", "Off", "OFF", "f", "F"]
+
+      def self.call(value)
+        case value
+        when *FALSE_VALUES then false
```

Comment:
The current state of typecasting leaves a lot to be desired, and certainly needs more work.  As it stands, switching to typecasting at write time instead of read solves the majority of the performance issues.  But there is still a lot more to do, see below.